packages/rs-dpp/src/data_contract/config/v0/mod.rs

```diff
@@ -1,81 +1,83 @@
-    pub documents_keep_history_contract_default: bool,
+    pub document_revisions_default: bool,
     /// Are documents in the contract mutable. This specifies whether the document can be
     /// changed or deleted. This is a default for all documents in the contract, but can be
     /// overridden by the document itself
-    pub documents_mutable_contract_default: bool,
+    pub documents_read_only_default: bool,
 }
 
 /// Trait representing getters for `DataContractConfigV0`
 pub trait DataContractConfigGettersV0 {
     /// Returns whether the contract can be deleted.
-    fn can_be_deleted(&self) -> bool;
+    fn is_contract_deletion_allowed(&self) -> bool;
 
     /// Returns whether the contract is read-only.
-    fn readonly(&self) -> bool;
+    fn is_contract_update_allowed(&self) -> bool;
 
     /// Returns whether the contract keeps history.
-    fn keeps_history(&self) -> bool;
+    fn keep_previous_contract_versions(&self) -> bool;
 
     /// Returns whether documents in the contract keep history by default.
-    fn documents_keep_history_contract_default(&self) -> bool;
+    fn document_revisions(&self) -> bool;
 
     /// Returns whether documents in the contract are mutable by default.
-    fn documents_mutable_contract_default(&self) -> bool;
+    fn documents_read_only(&self) -> bool;
 }
 
 /// Trait representing setters for `DataContractConfigV0`
 pub trait DataContractConfigSettersV0 {
     /// Sets whether the contract can be deleted.
-    fn set_can_be_deleted(&mut self, value: bool);
+    fn set_allow_contract_deletion(&mut self, value: bool);
 
     /// Sets whether the contract is read-only.
-    fn set_readonly(&mut self, value: bool);
+    fn set_allow_contract_update(&mut self, value: bool);
 
     /// Sets whether the contract keeps history.
-    fn set_keeps_history(&mut self, value: bool);
+    fn set_keep_previous_contract_versions(&mut self, value: bool);
 
     /// Sets whether documents in the contract keep history by default.
-    fn set_documents_keep_history_contract_default(&mut self, value: bool);
+    fn set_document_revisions_default(&mut self, value: bool);
 
     /// Sets whether documents in the contract are mutable by default.
-    fn set_documents_mutable_contract_default(&mut self, value: bool);
+    fn set_documents_read_only_default(&mut self, value: bool);
 }
 
 impl std::default::Default for DataContractConfigV0 {
     fn default() -> Self {
         DataContractConfigV0 {
-            can_be_deleted: DEFAULT_CONTRACT_CAN_BE_DELETED,
-            readonly: !DEFAULT_CONTRACT_MUTABILITY,
-            keeps_history: DEFAULT_CONTRACT_KEEPS_HISTORY,
-            documents_keep_history_contract_default: DEFAULT_CONTRACT_DOCUMENTS_KEEPS_HISTORY,
-            documents_mutable_contract_default: DEFAULT_CONTRACT_DOCUMENT_MUTABILITY,
+            allow_contract_deletion: DEFAULT_CONTRACT_CAN_BE_DELETED,
+            allowe_contract_update: !DEFAULT_CONTRACT_MUTABILITY,
```

Comment:
allow_contract update is the opposite of readonly so it should be allow_contract_update: DEFAULT_CONTRACT_MUTABILITY,